### PR TITLE
feat(docs): add guide for `NSQuitAlwaysKeepsWindow` setting

### PR DESCRIPTION
**Pull request by romen**
Add documentation for the NSQuitAlwaysKeepsWindow setting in macOS.

Includes instructions for setting, reading, and resetting the value.

This is my first contribution here, please provide feedback to improve it.

### CODE REVIEW

**Comment by yannbertrand** [comment]
Hello @romen thanks for the contribution :) it looks like there is small formatting issue, please run `npm run lint -- --fix` and amend the commit.

File: docs/misc/nsquitalwayskeepwindows.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  - - meta
+    - property: 'og:description'
+      content: When enabled, open documents and windows will be restored when you re-open an application.
+---
+
+# Keep windows when quitting an application
+
+When enabled, open documents and windows will be restored when you re-open an application.
+This setting is the inverse of the option accessible through
+System Preferences > Desktop & Dock > Close windows when quitting an application.
```
Comment (yannbertrand) [inline docs/misc/nsquitalwayskeepwindows.md:17]: Please move this part to a "Set value from UI" section at the end of the page. Take a look at [an example](https://github.com/yannbertrand/macos-defaults/blob/b4cc05a76c9459d389b8916e240a1337962045bc/docs/keyboard/applekeyboarduimode.md?plain=1#L60).

File: docs/misc/nsquitalwayskeepwindows.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+Requires logging out and in again before working.
```
Comment (yannbertrand) [inline docs/misc/nsquitalwayskeepwindows.md:18]:
```suggestion

⚠️ Restarting the Mac or closing the session is necessary to take the changes into effect.
```

File: docs/misc/nsquitalwayskeepwindows.md (hunk 1 above)
Comment (romen) [reply docs/misc/nsquitalwayskeepwindows.md]: Done, see 6afc85c488a0cfaab6b1820ee9674a820f7fae8e

**Comment by romen** [comment]
> Hello @romen thanks for the contribution :) it looks like there is small formatting issue, please run `npm run lint -- --fix` and amend the commit.

Thank you for the feedback!

After applying your suggestions and adding the _Set value from UI_ section, `npm run lint` shows all green here!

**Review by yannbertrand — APPROVED** [review]
Thanks :)

**Pull request merged**